packages/devtools_app/lib/src/primitives/utils.dart

```diff
@@ -550,52 +551,52 @@ class TimeRange {
 
   final bool singleAssignment;
 
-  Duration get start => _start;
+  Duration? get start => _start;
 
-  Duration _start;
+  Duration? _start;
 
-  set start(Duration value) {
+  set start(Duration? value) {
```

Comment:
non-nullable

Reply:
The fields _start and _end in this class are designed to be nullable. dart does not allow setter to be of different type.